json.spec

```diff
@@ -1,16 +1,21 @@
-### RPM external json 3.10.2
+### RPM external json 3.11.3
 ## NOCOMPILER
 
-Source: https://github.com/nlohmann/%{n}/releases/download/v%{realversion}/include.zip
+Source: https://github.com/nlohmann/json/archive/refs/tags/v%{realversion}.tar.gz
 
 %prep
-%setup -c
+%setup -n %{n}-%{realversion}
+sed -i -e 's|JSON_BuildTests_INIT ON|JSON_BuildTests_INIT OFF|' CMakeLists.txt
```

Comment:
I think you can just add `-DJSON_BuildTests=OFF` to the cmake line for the same effect (but more robust against future changes)

Reply:
thanks @sethrj